resources/public/api/conf/1.0/application.yaml

```diff
@@ -0,0 +1,385 @@
+    post:
+      tags:
+      - Arrivals
+      summary: Inject a fake NCTS Arrival Message
+      description: >
+        Inject a message type into the test NCTS database for a given Arrival Movement.
+      operationId: postFakeNCTSArrivalMessage
+      parameters:
+        - $ref: '#/components/parameters/contentTypeHeader'
+        - in: path
+          name: arrivalId
+          description: The ID specifying the Arrival
+          required: true
+          style: simple
+          schema:
+            type: string
+            example: '1'
+      requestBody:
+        description: JSON with message type
+        content:
+          application/json:
+            schema:
+              allOf:
+              - $ref: '#/components/schemas/TestCustomsTransitsMovementsArrivalsMessagesRequest'
+              - description: JSON with message type
+                example:
+                  message:
+                    messageType: IEXXX
+            example:
+              message:
+                messageType: IEXXX
+        required: true
+      responses:
+        '201':
+          description: When a test message has been successfully injected.
+          content:
+            application/json:
+              schema:
+                allOf:
+                - $ref: '#/components/schemas/TestCustomsTransitsMovementsArrivalsMessagesResponse'
+                - description: JSON Payload
+                  example:
+                    _links:
+                      self:
+                        href: /customs/transits/movements/arrivals/1/messages/2
+                      arrival:
+                        href: /customs/transits/movements/arrivals/1
+                    arrivalId: '1'
+                    messageId: '2'
+                    messageType: IEXXX
+                    body: <CCXXX>...</CCXXX>
+              example:
+                _links:
+                  self:
+                    href: /customs/transits/movements/arrivals/1/messages/2
+                  arrival:
+                    href: /customs/transits/movements/arrivals/1
+                arrivalId: '1'
+                messageId: '2'
+                messageType: IEXXX
+                body: <CCXXX>...</CCXXX>
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: '#/components/schemas/errorResponse'
+                  - description: The supplied payload was invalid.
+                    example:
+                      statusCode: 400
+              example:
+                statusCode: 400
+        '404':
+          description: Not Found
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: '#/components/schemas/errorResponse'
+                  - description: When the Arrival Movement does not exist, has been archived or is not available to the EORI number.

```

Comment:
"When" is awkward wording and prefer lowercase.

Suggestion:
Supplied arrival not found or does not exist or has been archived or is not available to the EORI number.